src/App.js

```diff
@@ -3,6 +3,7 @@ import './App.css';
 import Search from './Search.js';
 import { tvShow, spinOff, episodes } from './Data.js';
 import Carousel from './Carousel.js';
+import Watchlist from './Watchlist.js';
```

Comment:
This looks great. Everything appears to be working correctly and I have no suggestions for changes. It looks like what we set out to do at this point.